tools/add_triage_signature.py

```diff
@@ -50,15 +53,27 @@ def format_description(failure_data):
 def days_ago(datestr):
     try:
         return (datetime.now() - dateutil.parser.isoparse(datestr)).days
-    except:
+    except Exception:
```

Comment:
pylint doesn't like `except:` either:
![image](https://user-images.githubusercontent.com/10882062/111301631-1bb62580-865b-11eb-80c3-b54d4ede2da9.png)

Reply:
why won't we just raise here?
or catch more specific exception types here

Reply:
>why won't we just raise here?

We don't raise because we want to keep going. We don't care about problems in a single ticket.

> or catch more specific exception types here

That's definitely a better way to do things. I guess in this case we didn't want to bother and look at all the possible exceptions that might be raised in each case. In this PR I just improved it a little bit - `except:` wasn't letting me to `ctrl+c` to close the script because `except:` covers `KeyboardInterrupt` while `except Exception:` doesn't